manifests/config.pp

```diff
@@ -5,6 +5,7 @@
   $manage_configs_directory = true,
   $manage_dle               = false,
   $configs_source           = 'amanda/server',
+  $manage_configs_source    = true,
```

Comment:
please add a datatype to the parameter

Reply:
Just new parameters? 
I have no problem with giving all the parameters a make-over.

Reply:
Sure thing. I am away from my build environment for about a another week, but will get right on it then..

Reply:
Is there anything else preventing this PR?